### PR TITLE
Add Alico48

### PR DESCRIPTION
**Pull request by marby3**
I'll add the keyboard I made.
I made a keyboard that uses three TRRS cables.
It uses only one ProMicro.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/alico48/config.h [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/alico48/config.h:84]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/alico48/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![alico48](imgur.com image replace me!)
+
+The Alico48 is a split keyboard that uses only one ProMicro.
+
+* Keyboard Maintainer: [marby](https://github.com/marby3)
+* Hardware Supported: Alico48 PCB, ProMicro
+* Hardware Availability: *Links to where you can find this hardware*l
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/alico48/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/alico48/rules.mk [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = yes       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (drashna) [inline keyboards/alico48/rules.mk:29]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/alico48/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+    make alico48:default
+
+Flashing example for this keyboard:
+
+    make alico48:default:avrdude
```
Comment (fauxpark) [inline keyboards/alico48/readme.md:17]:
```suggestion
    make alico48:default:flash
```

File: keyboards/alico48/info.json [hunk 5]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (fauxpark) [inline keyboards/alico48/info.json:1]: This file is incorrect:
image

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by marby3** [comment]
I'm not very good with Github, were the changes appropriate?

File: keyboards/alico48/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+EXTRAKEY_ENABLE = no        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/alico48/rules.mk:21]:
```suggestion
```

File: keyboards/alico48/info.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    "keyboard_name": "alico48",
+    "url": "",
+    "maintainer": "marby3",
+    "width": 16,
+    "height": 11,
```
Comment (fauxpark) [inline keyboards/alico48/info.json:6]:
```suggestion
```

File: keyboards/alico48/readme.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
+# alico48
+
+![alico48](imgur.com image replace me!)
```
Comment (fauxpark) [inline keyboards/alico48/readme.md:3]: Please fill this out.

File: keyboards/alico48/readme.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+The Alico48 is a split keyboard that uses only one ProMicro.
+
+* Keyboard Maintainer: [marby](https://github.com/marby3)
+* Hardware Supported: Alico48 PCB, ProMicro
+* Hardware Availability: *Links to where you can find this hardware*l
```
Comment (fauxpark) [inline keyboards/alico48/readme.md:9]: As above.

File: keyboards/alico48/info.json (hunk 5 above)
Comment (fauxpark) [reply keyboards/alico48/info.json:1]: Still not correct. Please use the Configurator to check the positioning and ordering of the info.json layout matches the LAYOUT macro, by pressing Ctrl+Shift+I and turning on Fast Input. When you generate the info.json, don't use rotation either.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**